### ISSUE TITLE
Invalid CSS stylesheet

### ISSUE THREAD

**Issue by Dibbyo456** [better-studio/better-amp#24]
1) Invalid CSS stylesheet = http://prntscr.com/e82oe3

2) isn't the "/amp" supposed to be the at the end of the URL rather than middle?

**Comment by aliaghdam** [comment]
Hi dear, 

1. Please make sure your site cache is clear or give me the link of that page

2. No BetterAMP is about full AMP version of site because of this it can be only in beginning of permalink.

<strong>Better</strong>Studio, Ali

**Comment by Dibbyo456** [comment]
Hi, I figured out the Issue, I was using "WP Rocket" caching plugin's **Google Fonts Minification enabled**.
After disabling the Google Fonts Minification, the error went way, it gave me a full PASS.

But I don't understand when I use this plugin [AMP for WP - Accelerated Mobile Pages ](https://wordpress.org/plugins/accelerated-mobile-pages/), It does not give me any error even all the minifications are enabled.

2. I did not understood what you said about "BetterAMP is about full AMP version of site", what's the difference ? I would love to see a good explanation if you don't bother. 👍
